### DISCUSSION TITLE
Red Electribe Card Error

### DISCUSSION THREAD

**Discussion by failinghackerman** [bangcorrupt/hacktribe#299] [Q&A] [answered]
Ok so I had my Red hacked for a long time and decided to return to factory firmware (by pressing Factory Reset too..if this helps) for a bit but whenever I try to do something with the SD (either import samples or change firmware) it just pops an "ERROR Card Error" while still in hacktribe firmware. Also if my SD is not formatted it appears as corrupted on my PC but if formatted it works "normally" until it gets back in the Electribe. I tried several SD cards it's always the same. I tried most of the tutorials here and nothing works. I have no idea of code although I'm willing to learn to save this machine.
Any help would be greatly appreciated.
Thank you for your time and amazing work.

**Comment by bangcorrupt** [comment, marked as answer]
Do #147, #149 and #298 help?

There were some SD card bugs in older versions of Hacktribe.  If you have new IFX delays at the end of the list then these should be fixed.  I would first eliminate as many bugs as possible by updating to the latest Hacktribe.
